src/wristpy/processing/analytics.py

```diff
@@ -122,11 +123,15 @@ def _spt_window(
         """
         anglez_abs_diff = self._compute_abs_diff_mean_anglez(anglez_data)
         anglez_median_long_epoch = computations.moving_median(anglez_abs_diff, 300)
```

Comment:
In general, if you use numbers as input for a function without a variable name, I'd recommend using a keyword argument e.g. `moving_median(anglez_abs_diff, epoch_length=300)`. As an aside, the same recommendation holds for True/False.

Reply:
The same goes for the numbers in fill_short_blocks. This should also make it a bit more evident that these are different units.